Liquibase and SQLite doesn't work with NoClassDefFoundError: org/slf4j/LoggerFactory error

### Search first

- [X] I searched and no similar issues were found

### Description

`liquibase --version`

```
Starting Liquibase at 08:26:33 (version 4.26.0 #1141 built at 2024-02-06 21:31+0000)
Liquibase Home: /opt/homebrew/Cellar/liquibase/4.26.0/libexec
Java Home /opt/homebrew/Cellar/openjdk/21.0.2/libexec/openjdk.jdk/Contents/Home (Version 21.0.2)
Libraries:
- internal/lib/commons-collections4.jar: Apache Commons Collections 4.4.0 By The Apache Software Foundation
- internal/lib/commons-lang3.jar: Apache Commons Lang 3.13.0 By The Apache Software Foundation
- internal/lib/commons-text.jar: Apache Commons Text 1.11.0 By The Apache Software Foundation
- internal/lib/h2.jar: H2 Database Engine 2.2.224 By H2 Group
- internal/lib/hsqldb.jar: HSQLDB 2.7.2 By The HSQL Development Group
- internal/lib/jaxb-api.jar: jaxb-api 2.3.1 By Oracle Corporation
- internal/lib/jaxb-core.jar: JAXB Core 4.0.4 By Eclipse Foundation
- internal/lib/jaxb-runtime.jar: JAXB Runtime 4.0.4 By Eclipse Foundation
- internal/lib/jaybird.jar: Jaybird 5.0.3.java8 (build: variant=jaybird tag=v5.0.3 date=202311241106) By Firebird project
- internal/lib/jcc.jar: IBM JCC JDBC 4 Driver 1.4.0 By IBM
- internal/lib/liquibase-commercial.jar: Commercial Liquibase Functionality 4.26.0 By Liquibase
- internal/lib/mariadb-java-client.jar: mariadb-java-client 3.3.2 By mariadb.com
- internal/lib/mssql-jdbc.jar: Microsoft JDBC Driver for SQL Server 12.4.1 By Microsoft Corporation
- internal/lib/ojdbc8.jar: JDBC 19.21.0.0.0 By Oracle Corporation
- internal/lib/opencsv.jar: opencsv 5.9.0
- internal/lib/picocli.jar: picocli 4.7.5 By Remko Popma
- internal/lib/postgresql.jar: PostgreSQL JDBC Driver 42.7.1 By PostgreSQL Global Development Group
- internal/lib/snakeyaml.jar: SnakeYAML 2.2.0
- internal/lib/snowflake-jdbc.jar: snowflake-jdbc 3.14.4
- internal/lib/sqlite-jdbc.jar: SQLite JDBC 3.44.1.0


Liquibase Version: 4.26.0
Liquibase Open Source 4.26.0 by Liquibase
```


### Steps To Reproduce

I am trying to do

`liquibase --changeLogFile=initial.sqlite.yaml --url=jdbc:sqlite:/liquibase/bot.db`

And getting such an error

```
Starting Liquibase at 08:28:23 (version 4.26.0 #1141 built at 2024-02-06 21:31+0000)
Liquibase Version: 4.26.0
Liquibase Open Source 4.26.0 by Liquibase

Unexpected error running Liquibase: Could not initialize class org.sqlite.JDBC
  - Caused by: Exception java.lang.NoClassDefFoundError: org/slf4j/LoggerFactory [in thread "main"]

For more information, please use the --log-level flag
```

### Expected/Desired Behavior

Expected behavior is to apply my files

```
databaseChangeLog:
    - include:
        file: changelog/base.yaml
```

and

```
databaseChangeLog:
  - changeSet:
      id: 1
      author: mossad
      changes:
        - createTable:
            tableName: allowed_users
            columns:
              - column:
                  name: user_id
                  type: int
                  constraints:
                    primaryKey: true
                    nullable: false
```


### Liquibase Version

4.26.0

### Database Vendor & Version

SQLite

### Liquibase Integration

CLI

### Liquibase Extensions

_No response_

### OS and/or Infrastructure Type/Provider

macOS

### Additional Context

**Please help**

1. I am running `update` command
2. I am running a command against an empty file, so there is not actually a version for SQLite itself
3. Here is debug

```
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for directory /Users/mossad/Y/danilevech/mentor/comrade-chatgpt-bot/liquibase
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.searchPath found
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for file /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/liquibase-core.jar
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for directory /opt/homebrew/Cellar/liquibase/4.26.0/libexec/lib
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for file /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/jaybird.jar
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for file /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/ojdbc8.jar
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for file /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/snakeyaml.jar
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for file /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/snowflake-jdbc.jar
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for file /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/picocli.jar
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for file /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/jaxb-runtime.jar
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for file /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/jaxb-api.jar
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for file /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/jaxb-core.jar
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for file /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/hsqldb.jar
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for file /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/mssql-jdbc.jar
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for file /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/h2.jar
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for file /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/mariadb-java-client.jar
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for file /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/liquibase-commercial.jar
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for file /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/commons-lang3.jar
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for file /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/postgresql.jar
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for file /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/sqlite-jdbc.jar
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for file /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/opencsv.jar
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for file /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/commons-text.jar
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for file /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/commons-collections4.jar
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for file /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/jcc.jar
[2024-03-28 02:53:24] FINE [liquibase.resource] Creating resourceAccessor for directory /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib
[2024-03-28 02:53:24] FINE [liquibase.resource] Overall search path: 
  - /Users/mossad/Y/danilevech/mentor/comrade-chatgpt-bot/liquibase
  - /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/liquibase-core.jar
  - /opt/homebrew/Cellar/liquibase/4.26.0/libexec/lib
  - /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/jaybird.jar
  - /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/ojdbc8.jar
  - /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/snakeyaml.jar
  - /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/snowflake-jdbc.jar
  - /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/picocli.jar
  - /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/jaxb-runtime.jar
  - /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/jaxb-api.jar
  - /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/jaxb-core.jar
  - /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/hsqldb.jar
  - /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/mssql-jdbc.jar
  - /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/h2.jar
  - /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/mariadb-java-client.jar
  - /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/liquibase-commercial.jar
  - /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/commons-lang3.jar
  - /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/postgresql.jar
  - /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/sqlite-jdbc.jar
  - /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/opencsv.jar
  - /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/commons-text.jar
  - /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/commons-collections4.jar
  - /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib/jcc.jar
  - /opt/homebrew/Cellar/liquibase/4.26.0/libexec/internal/lib
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.uiService found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.uiService is using the default value of CONSOLE
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.mirrorConsoleMessagesToLog found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.mirrorConsoleMessagesToLog is using the default value of true
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.shouldRun aka should.run found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.shouldRun is using the default value of true
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.showBanner found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.showBanner is using the default value of true
####################################################
##   _     _             _ _                      ##
##  | |   (_)           (_) |                     ##
##  | |    _  __ _ _   _ _| |__   __ _ ___  ___   ##
##  | |   | |/ _` | | | | | '_ \ / _` / __|/ _ \  ##
##  | |___| | (_| | |_| | | |_) | (_| \__ \  __/  ##
##  \_____/_|\__, |\__,_|_|_.__/ \__,_|___/\___|  ##
##              | |                               ##
##              |_|                               ##
##                                                ## 
##  Get documentation at docs.liquibase.com       ##
##  Get certified courses at learn.liquibase.com  ## 
##                                                ##
####################################################
Starting Liquibase at 02:53:24 (version 4.26.0 #1141 built at 2024-02-06 21:31+0000)
[2024-03-28 02:53:24] INFO [liquibase.ui] ####################################################
##   _     _             _ _                      ##
##  | |   (_)           (_) |                     ##
##  | |    _  __ _ _   _ _| |__   __ _ ___  ___   ##
##  | |   | |/ _` | | | | | '_ \ / _` / __|/ _ \  ##
##  | |___| | (_| | |_| | | |_) | (_| \__ \  __/  ##
##  \_____/_|\__, |\__,_|_|_.__/ \__,_|___/\___|  ##
##              | |                               ##
##              |_|                               ##
##                                                ## 
##  Get documentation at docs.liquibase.com       ##
##  Get certified courses at learn.liquibase.com  ## 
##                                                ##
####################################################
Starting Liquibase at 02:53:24 (version 4.26.0 #1141 built at 2024-02-06 21:31+0000)
Liquibase Version: 4.26.0
[2024-03-28 02:53:24] INFO [liquibase.ui] Liquibase Version: 4.26.0
Liquibase Open Source 4.26.0 by Liquibase
[2024-03-28 02:53:24] INFO [liquibase.ui] Liquibase Open Source 4.26.0 by Liquibase
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.monitorPerformance found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.monitorPerformance is using the default value of false
[2024-03-28 02:53:24] FINE [liquibase.integration] Performance monitoring disabled
[2024-03-28 02:53:24] INFO [liquibase.integration] Starting command execution.
[2024-03-28 02:53:24] FINE [liquibase.command] Found ProUpdateCommandStep override for UpdateCommandStep! Using ProUpdateCommandStep in pipeline.
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.outputFile found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.strict found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.strict is using the default value of false
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.changeExecListener found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.changeExecListener found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.changeExecListenerClass found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.changeExecListenerClass found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.changeExecListenerPropertiesFile found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.changeExecListenerPropertiesFile found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.changelogFile found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Found 'liquibase.command.changelogFile' configuration of 'initial.sqlite.yaml'
    command argument '--changelogFile' of 'initial.sqlite.yaml'
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.changelogParameters found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.changelogParameters found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.contextFilter aka liquibase.command.update.contexts found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.contextFilter aka liquibase.command.contexts found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.database found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.database found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.databaseChangelog found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.databaseChangelog found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.defaultCatalogName found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.defaultCatalogName found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.defaultSchemaName found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.defaultSchemaName found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.driver found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.driver found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.driverPropertiesFile found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.driverPropertiesFile found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.labelFilter aka liquibase.command.update.labels found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.labelFilter aka liquibase.command.labels found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.password found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.password found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.reportEnabled aka liquibase.command.update.reportsEnabled found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.reportEnabled aka liquibase.command.reportsEnabled found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.reportFormat aka liquibase.command.update.reportsFormat found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.reportFormat aka liquibase.command.reportsFormat found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.reportName aka liquibase.command.update.reportsName found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.reportName aka liquibase.command.reportsName found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.reportPath aka liquibase.command.update.reportsPath found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.reportPath aka liquibase.command.reportsPath found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.rollbackOnError found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.command.update.rollbackOnError is using the default value of false
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.rollbackOnError found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.command.rollbackOnError is using the default value of false
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.showSummary found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.command.update.showSummary is using the default value of SUMMARY
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.showSummary found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.command.showSummary is using the default value of SUMMARY
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.showSummaryOutput found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.command.update.showSummaryOutput is using the default value of ALL
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.showSummaryOutput found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.command.showSummaryOutput is using the default value of ALL
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.skipDatabaseStep found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.command.update.skipDatabaseStep is using the default value of false
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.skipDatabaseStep found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.command.skipDatabaseStep is using the default value of false
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.updateNullChecksums found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.command.update.updateNullChecksums is using the default value of false
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.updateNullChecksums found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.command.updateNullChecksums is using the default value of false
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.url found
[2024-03-28 02:53:24] FINE [liquibase.servicelocator] Loaded liquibase.database.jvm.ConnectionPatterns instance liquibase.database.jvm.JdbcConnectionPatterns
[2024-03-28 02:53:24] FINE [liquibase.configuration] Found 'liquibase.command.url' configuration of 'jdbc:sqlite:/liquibase/bot.db'
    command argument '--url' of '*****'
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.update.username found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.command.username found
[2024-03-28 02:53:24] FINE [liquibase.command] Pipeline for command 'update: liquibase.command.core.helpers.DbUrlConnectionArgumentsCommandStep then liquibase.command.core.helpers.ShowSummaryArgument then com.datical.liquibase.ext.command.helpers.RollbackOnErrorArgument then com.datical.liquibase.ext.command.helpers.VisibleReportCommandArguments then liquibase.command.core.helpers.DbUrlConnectionCommandStep then liquibase.command.core.helpers.DatabaseChangelogCommandStep then liquibase.command.core.helpers.ChangeExecListenerCommandStep then com.datical.liquibase.ext.command.ProUpdateCommandStep
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.outputFileEncoding found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.outputFileEncoding is using the default value of UTF-8
[2024-03-28 02:53:24] FINE [liquibase.command] Writing output with encoding 'UTF-8'
[2024-03-28 02:53:24] FINE [liquibase.command] Executing internal command dbUrlConnectionArguments
[2024-03-28 02:53:24] FINE [liquibase.command] Executing internal command showSummary
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.command.update.showSummary is using the default value of SUMMARY
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.command.showSummary is using the default value of SUMMARY
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.command.update.showSummaryOutput is using the default value of ALL
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.command.showSummaryOutput is using the default value of ALL
[2024-03-28 02:53:24] FINE [liquibase.command] Executing internal command rollbackOnError
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.command.update.rollbackOnError is using the default value of false
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.command.rollbackOnError is using the default value of false
[2024-03-28 02:53:24] FINE [liquibase.command] Executing internal command reportvisible
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.reports.enabled aka liquibase.report.enabled found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.reports.enabled is using the default value of false
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.reports.name aka liquibase.report.name found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.reports.name is using the default value of report-dd-MMM-yyyy-HHmmss
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.reports.path aka liquibase.report.path found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.reports.path is using the default value of .
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.reports.format aka liquibase.report.format found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.reports.format is using the default value of HTML
[2024-03-28 02:53:24] FINE [liquibase.command] Executing internal command dbUrlConnectionCommandStep
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.command.update.skipDatabaseStep is using the default value of false
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.command.skipDatabaseStep is using the default value of false
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.liquibaseCatalogName aka liquibase.catalogName found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.liquibaseSchemaName aka liquibase.schemaName found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.databaseClass found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.propertyProviderClass found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.liquibaseTablespaceName aka liquibase.databaseChangeLogTablespaceName aka liquibase.liquibaseTableSpaceName found
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.databaseChangelogLockTableName aka liquibase.databaseChangeLogLockTableName found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.databaseChangelogLockTableName is using the default value of DATABASECHANGELOGLOCK
[2024-03-28 02:53:24] FINE [liquibase.configuration] No configuration value for liquibase.databaseChangelogTableName aka liquibase.databaseChangeLogTableName found
[2024-03-28 02:53:24] FINE [liquibase.configuration] Configuration liquibase.databaseChangelogTableName is using the default value of DATABASECHANGELOG
[2024-03-28 02:53:24] FINE [liquibase.database] Properties:
[2024-03-28 02:53:24] INFO [liquibase.command] Command execution complete
[2024-03-28 02:53:24] SEVERE [liquibase.integration] Could not initialize class org.sqlite.JDBC
  - Caused by: Exception java.lang.NoClassDefFoundError: org/slf4j/LoggerFactory [in thread "main"]
java.lang.NoClassDefFoundError: Could not initialize class org.sqlite.JDBC
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:534)
	at java.base/java.lang.Class.forName(Class.java:513)
	at liquibase.database.DatabaseFactory.loadDriver(DatabaseFactory.java:299)
	at liquibase.database.DatabaseFactory.openConnection(DatabaseFactory.java:225)
	at liquibase.database.DatabaseFactory.openConnection(DatabaseFactory.java:188)
	at liquibase.database.DatabaseFactory.openDatabase(DatabaseFactory.java:153)
	at liquibase.command.core.helpers.AbstractDatabaseConnectionCommandStep.createDatabaseObject(AbstractDatabaseConnectionCommandStep.java:73)
	at liquibase.command.core.helpers.DbUrlConnectionCommandStep.obtainDatabase(DbUrlConnectionCommandStep.java:108)
	at liquibase.command.core.helpers.DbUrlConnectionCommandStep.run(DbUrlConnectionCommandStep.java:80)
	at liquibase.command.CommandScope.execute(CommandScope.java:217)
	at liquibase.integration.commandline.CommandRunner.call(CommandRunner.java:55)
	at liquibase.integration.commandline.CommandRunner.call(CommandRunner.java:24)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2041)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
	at picocli.CommandLine.execute(CommandLine.java:2170)
	at liquibase.integration.commandline.LiquibaseCommandLine.lambda$execute$2(LiquibaseCommandLine.java:390)
	at liquibase.Scope.child(Scope.java:195)
	at liquibase.Scope.child(Scope.java:171)
	at liquibase.integration.commandline.LiquibaseCommandLine.lambda$execute$3(LiquibaseCommandLine.java:365)
	at liquibase.Scope.child(Scope.java:195)
	at liquibase.Scope.child(Scope.java:171)
	at liquibase.integration.commandline.LiquibaseCommandLine.execute(LiquibaseCommandLine.java:362)
	at liquibase.integration.commandline.LiquibaseCommandLine.main(LiquibaseCommandLine.java:102)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at liquibase.integration.commandline.LiquibaseLauncher.main(LiquibaseLauncher.java:116)
Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.NoClassDefFoundError: org/slf4j/LoggerFactory [in thread "main"]
	at org.sqlite.JDBC.<clinit>(JDBC.java:26)
	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized0(Native Method)
	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized(Unsafe.java:1160)
	at java.base/jdk.internal.reflect.MethodHandleAccessorFactory.ensureClassInitialized(MethodHandleAccessorFactory.java:300)
	at java.base/jdk.internal.reflect.MethodHandleAccessorFactory.newConstructorAccessor(MethodHandleAccessorFactory.java:103)
	at java.base/jdk.internal.reflect.ReflectionFactory.newConstructorAccessor(ReflectionFactory.java:200)
	at java.base/java.lang.reflect.Constructor.acquireConstructorAccessor(Constructor.java:549)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
	at java.base/java.util.ServiceLoader$ProviderImpl.newInstance(ServiceLoader.java:789)
	at java.base/java.util.ServiceLoader$ProviderImpl.get(ServiceLoader.java:729)
	at java.base/java.util.ServiceLoader$3.next(ServiceLoader.java:1403)
	at java.sql/java.sql.DriverManager$2.run(DriverManager.java:619)
	at java.sql/java.sql.DriverManager$2.run(DriverManager.java:599)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:319)
	at java.sql/java.sql.DriverManager.ensureDriversInitialized(DriverManager.java:599)
	at java.sql/java.sql.DriverManager.getDrivers(DriverManager.java:427)
	at liquibase.database.core.DerbyDatabase.determineDriverVersion(DerbyDatabase.java:203)
	at liquibase.database.core.DerbyDatabase.<init>(DerbyDatabase.java:31)
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
	at java.base/java.util.ServiceLoader$ProviderImpl.newInstance(ServiceLoader.java:789)
	at java.base/java.util.ServiceLoader$ProviderImpl.get(ServiceLoader.java:729)
	at java.base/java.util.ServiceLoader$3.next(ServiceLoader.java:1403)
	at liquibase.servicelocator.StandardServiceLocator.findInstances(StandardServiceLocator.java:24)
	at liquibase.database.DatabaseFactory.<init>(DatabaseFactory.java:39)
	at liquibase.database.DatabaseFactory.getInstance(DatabaseFactory.java:50)
	at liquibase.change.core.AddForeignKeyConstraintChange.createSupportedDatabasesMetaData(AddForeignKeyConstraintChange.java:52)
	at liquibase.change.AbstractChange.createChangeParameterMetadata(AbstractChange.java:200)
	at liquibase.change.AbstractChange.createChangeMetaData(AbstractChange.java:94)
	at liquibase.change.ChangeFactory.lambda$getChangeMetaData$0(ChangeFactory.java:56)
	at java.base/java.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:1740)
	at liquibase.change.ChangeFactory.getChangeMetaData(ChangeFactory.java:56)
	at liquibase.change.ChangeFactory.getDefinedChanges(ChangeFactory.java:92)
	at liquibase.command.core.helpers.AbstractChangelogCommandStep.supportedReplaceIfExistsTypes(AbstractChangelogCommandStep.java:59)
	at liquibase.command.core.helpers.AbstractChangelogCommandStep.<clinit>(AbstractChangelogCommandStep.java:28)
	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized0(Native Method)
	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized(Unsafe.java:1160)
	at java.base/jdk.internal.reflect.MethodHandleAccessorFactory.ensureClassInitialized(MethodHandleAccessorFactory.java:300)
	at java.base/jdk.internal.reflect.MethodHandleAccessorFactory.newConstructorAccessor(MethodHandleAccessorFactory.java:103)
	at java.base/jdk.internal.reflect.ReflectionFactory.newConstructorAccessor(ReflectionFactory.java:200)
	at java.base/java.lang.reflect.Constructor.acquireConstructorAccessor(Constructor.java:549)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
	at java.base/java.util.ServiceLoader$ProviderImpl.newInstance(ServiceLoader.java:789)
	at java.base/java.util.ServiceLoader$ProviderImpl.get(ServiceLoader.java:729)
	at java.base/java.util.ServiceLoader$3.next(ServiceLoader.java:1403)
	at liquibase.servicelocator.StandardServiceLocator.findInstances(StandardServiceLocator.java:24)
	at liquibase.command.CommandFactory.findAllInstances(CommandFactory.java:243)
	at liquibase.command.CommandFactory.getCommands(CommandFactory.java:162)
	at liquibase.integration.commandline.LiquibaseCommandLine.getCommands(LiquibaseCommandLine.java:1110)
	at liquibase.integration.commandline.LiquibaseCommandLine.buildPicoCommandLine(LiquibaseCommandLine.java:216)
	at liquibase.integration.commandline.LiquibaseCommandLine.<init>(LiquibaseCommandLine.java:192)
	at liquibase.integration.commandline.LiquibaseCommandLine.main(LiquibaseCommandLine.java:100)
	... 3 more


Unexpected error running Liquibase: Could not initialize class org.sqlite.JDBC
  - Caused by: Exception java.lang.NoClassDefFoundError: org/slf4j/LoggerFactory [in thread "main"]

java.lang.NoClassDefFoundError: Could not initialize class org.sqlite.JDBC
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:534)
	at java.base/java.lang.Class.forName(Class.java:513)
	at liquibase.database.DatabaseFactory.loadDriver(DatabaseFactory.java:299)
	at liquibase.database.DatabaseFactory.openConnection(DatabaseFactory.java:225)
	at liquibase.database.DatabaseFactory.openConnection(DatabaseFactory.java:188)
	at liquibase.database.DatabaseFactory.openDatabase(DatabaseFactory.java:153)
	at liquibase.command.core.helpers.AbstractDatabaseConnectionCommandStep.createDatabaseObject(AbstractDatabaseConnectionCommandStep.java:73)
	at liquibase.command.core.helpers.DbUrlConnectionCommandStep.obtainDatabase(DbUrlConnectionCommandStep.java:108)
	at liquibase.command.core.helpers.DbUrlConnectionCommandStep.run(DbUrlConnectionCommandStep.java:80)
	at liquibase.command.CommandScope.execute(CommandScope.java:217)
	at liquibase.integration.commandline.CommandRunner.call(CommandRunner.java:55)
	at liquibase.integration.commandline.CommandRunner.call(CommandRunner.java:24)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2041)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
	at picocli.CommandLine.execute(CommandLine.java:2170)
	at liquibase.integration.commandline.LiquibaseCommandLine.lambda$execute$2(LiquibaseCommandLine.java:390)
	at liquibase.Scope.child(Scope.java:195)
	at liquibase.Scope.child(Scope.java:171)
	at liquibase.integration.commandline.LiquibaseCommandLine.lambda$execute$3(LiquibaseCommandLine.java:365)
	at liquibase.Scope.child(Scope.java:195)
	at liquibase.Scope.child(Scope.java:171)
	at liquibase.integration.commandline.LiquibaseCommandLine.execute(LiquibaseCommandLine.java:362)
	at liquibase.integration.commandline.LiquibaseCommandLine.main(LiquibaseCommandLine.java:102)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at liquibase.integration.commandline.LiquibaseLauncher.main(LiquibaseLauncher.java:116)
Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.NoClassDefFoundError: org/slf4j/LoggerFactory [in thread "main"]
	at org.sqlite.JDBC.<clinit>(JDBC.java:26)
	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized0(Native Method)
	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized(Unsafe.java:1160)
	at java.base/jdk.internal.reflect.MethodHandleAccessorFactory.ensureClassInitialized(MethodHandleAccessorFactory.java:300)
	at java.base/jdk.internal.reflect.MethodHandleAccessorFactory.newConstructorAccessor(MethodHandleAccessorFactory.java:103)
	at java.base/jdk.internal.reflect.ReflectionFactory.newConstructorAccessor(ReflectionFactory.java:200)
	at java.base/java.lang.reflect.Constructor.acquireConstructorAccessor(Constructor.java:549)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
	at java.base/java.util.ServiceLoader$ProviderImpl.newInstance(ServiceLoader.java:789)
	at java.base/java.util.ServiceLoader$ProviderImpl.get(ServiceLoader.java:729)
	at java.base/java.util.ServiceLoader$3.next(ServiceLoader.java:1403)
	at java.sql/java.sql.DriverManager$2.run(DriverManager.java:619)
	at java.sql/java.sql.DriverManager$2.run(DriverManager.java:599)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:319)
	at java.sql/java.sql.DriverManager.ensureDriversInitialized(DriverManager.java:599)
	at java.sql/java.sql.DriverManager.getDrivers(DriverManager.java:427)
	at liquibase.database.core.DerbyDatabase.determineDriverVersion(DerbyDatabase.java:203)
	at liquibase.database.core.DerbyDatabase.<init>(DerbyDatabase.java:31)
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
	at java.base/java.util.ServiceLoader$ProviderImpl.newInstance(ServiceLoader.java:789)
	at java.base/java.util.ServiceLoader$ProviderImpl.get(ServiceLoader.java:729)
	at java.base/java.util.ServiceLoader$3.next(ServiceLoader.java:1403)
	at liquibase.servicelocator.StandardServiceLocator.findInstances(StandardServiceLocator.java:24)
	at liquibase.database.DatabaseFactory.<init>(DatabaseFactory.java:39)
	at liquibase.database.DatabaseFactory.getInstance(DatabaseFactory.java:50)
	at liquibase.change.core.AddForeignKeyConstraintChange.createSupportedDatabasesMetaData(AddForeignKeyConstraintChange.java:52)
	at liquibase.change.AbstractChange.createChangeParameterMetadata(AbstractChange.java:200)
	at liquibase.change.AbstractChange.createChangeMetaData(AbstractChange.java:94)
	at liquibase.change.ChangeFactory.lambda$getChangeMetaData$0(ChangeFactory.java:56)
	at java.base/java.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:1740)
	at liquibase.change.ChangeFactory.getChangeMetaData(ChangeFactory.java:56)
	at liquibase.change.ChangeFactory.getDefinedChanges(ChangeFactory.java:92)
	at liquibase.command.core.helpers.AbstractChangelogCommandStep.supportedReplaceIfExistsTypes(AbstractChangelogCommandStep.java:59)
	at liquibase.command.core.helpers.AbstractChangelogCommandStep.<clinit>(AbstractChangelogCommandStep.java:28)
	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized0(Native Method)
	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized(Unsafe.java:1160)
	at java.base/jdk.internal.reflect.MethodHandleAccessorFactory.ensureClassInitialized(MethodHandleAccessorFactory.java:300)
	at java.base/jdk.internal.reflect.MethodHandleAccessorFactory.newConstructorAccessor(MethodHandleAccessorFactory.java:103)
	at java.base/jdk.internal.reflect.ReflectionFactory.newConstructorAccessor(ReflectionFactory.java:200)
	at java.base/java.lang.reflect.Constructor.acquireConstructorAccessor(Constructor.java:549)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
	at java.base/java.util.ServiceLoader$ProviderImpl.newInstance(ServiceLoader.java:789)
	at java.base/java.util.ServiceLoader$ProviderImpl.get(ServiceLoader.java:729)
	at java.base/java.util.ServiceLoader$3.next(ServiceLoader.java:1403)
	at liquibase.servicelocator.StandardServiceLocator.findInstances(StandardServiceLocator.java:24)
	at liquibase.command.CommandFactory.findAllInstances(CommandFactory.java:243)
	at liquibase.command.CommandFactory.getCommands(CommandFactory.java:162)
	at liquibase.integration.commandline.LiquibaseCommandLine.getCommands(LiquibaseCommandLine.java:1110)
	at liquibase.integration.commandline.LiquibaseCommandLine.buildPicoCommandLine(LiquibaseCommandLine.java:216)
	at liquibase.integration.commandline.LiquibaseCommandLine.<init>(LiquibaseCommandLine.java:192)
	at liquibase.integration.commandline.LiquibaseCommandLine.main(LiquibaseCommandLine.java:100)
	... 3 more

```

--- a/liquibase-dist/expected-distribution-contents.txt
+++ b/liquibase-dist/expected-distribution-contents.txt
@@ -78,6 +78,7 @@
 │       ├── opencsv.jar
 │       ├── picocli.jar
 │       ├── postgresql.jar
+│       ├── slf4j-api.jar
 │       ├── snakeyaml.jar
 │       ├── snowflake-jdbc.jar
 │       └── sqlite-jdbc.jar

--- a/liquibase-dist/src/main/assembly/assembly-bin-common.xml
+++ b/liquibase-dist/src/main/assembly/assembly-bin-common.xml
@@ -111,7 +111,6 @@
 
             <!-- some libraries lie about compile vs. runtime dependencies. Or we don't hit code that uses these. So exclude them manually. -->
             <excludes>
-                <exclude>org.slf4j:slf4j-api:jar:</exclude> <!-- from sqlite-jdbc -->
                 <exclude>org.antlr:antlr4-runtime:jar:</exclude>  <!-- from connector-api -->
                 <exclude>org.checkerframework:checker-qual:jar:</exclude> <!-- from postgresql -->
                 <exclude>commons-beanutils:commons-beanutils:jar:</exclude> <!-- from opencsv -->
@@ -167,7 +166,6 @@
 				<exclude>org.firebirdsql.jdbc:jaybird:</exclude> <!-- from internal/lib -->
 				<exclude>net.snowflake:snowflake-jdbc:</exclude> <!-- from internal/lib -->
 				<exclude>javax.resource:connector-api:</exclude> <!-- from internal/lib -->
-				<exclude>org.slf4j:slf4j-api:jar:</exclude> <!-- from sqlite-jdbc -->
 				<exclude>org.antlr:antlr4-runtime:jar:</exclude>  <!-- from connector-api -->
 				<exclude>org.checkerframework:checker-qual:jar:</exclude> <!-- from postgresql -->
 				<exclude>commons-beanutils:commons-beanutils:jar:</exclude> <!-- from opencsv -->

